pkgs/servers/mail/dovecot/default.nix

```diff
@@ -67,7 +69,8 @@ stdenv.mkDerivation rec {
     ++ lib.optional (stdenv.isDarwin) "--enable-static"
     ++ lib.optional withMySQL "--with-mysql"
     ++ lib.optional withPgSQL "--with-pgsql"
-    ++ lib.optional withSQLite "--with-sqlite";
+    ++ lib.optional withSQLite "--with-sqlite"
+    ++ lib.optional enableSASL "CFLAGS=-I${cyrus_sasl.dev}/include/sasl";
```

Comment:
Adding `cyrus_sasl` to `buildInputs` should have the same effect because of our c wrapper.

Reply:
```console
$ nix-shell -p cyrus_sasl --command 'echo $NIX_CFLAGS_COMPILE | grep sasl'
-isystem /nix/store/80rpfirmjs2zbckhk9srv6432jbzrpms-cyrus-sasl-2.1.26-dev/include -isystem /nix/store/80rpfirmjs2zbckhk9srv6432jbzrpms-cyrus-sasl-2.1.26-dev/include
```